MMM-COVID19.js

```diff
@@ -137,7 +141,7 @@ Module.register("MMM-COVID19", {
           deaths = globalStats["total_deaths"],
           newDeaths = globalStats["new_deaths"],
           totalRecovered = globalStats["total_recovered"],
-          activeCases = '';
+          activeCases = (cases && totalRecovered)?this.numberWithCommas(parseInt(cases.replace(",","")) - parseInt(totalRecovered.replace(",",""))):"";
```

Comment:
Could you please indent it properly? So we don't have long single lines
I would suggest you:

```
activeCases = (cases && totalRecovered) ? 
  this.numberWithCommas(parseInt(cases.replace(",","")) - parseInt(totalRecovered.replace(",","")))
  : "";

Reply:
done in 803276d